build-ci/platforms.rst

```diff
@@ -0,0 +1,33 @@
+###################
+Supported Platforms
+###################
+
+Introduction
+============
+
+The LSST Data Management software is required to be portable across many compute platforms ranging from high-performance computing centres to laptops (DMS-REQ-0308).
+The software will likely work on any Unix-like operating system but we limit our testing and support to a fixed set of operating systems.
+
+Patches will be accepted by developers who test on other operating systems.
```

Comment:
Perhaps "may"? When we receive a bazillion line patch set updating our code to work on Xenix, I doubt we're going to accept it.